app/helpers/application_helper.rb

```diff
@@ -66,7 +66,7 @@ def title(page_title)
   end
 
   def ga_scroll_ids(ids)
-    content_for(:ga_scroll_ids) { ids.join(',') }
+    content_for(:ga_scroll_ids) { ids.join(',')+',' }
```

Comment:
`content_for` concatenates everything sent to it into a single string, as mentioned in http://apidock.com/rails/v4.2.1/ActionView/Helpers/CaptureHelper/content_for, so it's not acting like a regular variable otherwise we should just send it many ids arrays and join them all together in application.html.erb